### ISSUE TITLE
Lighthouse accessibility error

### ISSUE THREAD

**Issue by martinverbic** [TryGhost/Starter#64]
It appears that Lighthouse is unable to properly analyze the Starter theme. The score for each segment is displayed, except for the "accessibility" part, where there appears to be an error (see screenshot). Other themes from Ghost don't appear to have this issue.

Any ideas how to solve this and what the problem is?

<img width="960" alt="Screen Shot 2023-05-14 at 11 55 02" src="https://github.com/TryGhost/Starter/assets/55486863/9b8ca1d8-587e-474d-a327-7a624df3fcaf">
<img width="973" alt="Screen Shot 2023-05-14 at 11 55 09" src="https://github.com/TryGhost/Starter/assets/55486863/bf6629b2-af44-42a3-81d8-546db8c7e0fe">
